src/Testing/InteractsWithMail.php

```diff
@@ -147,4 +147,9 @@ protected function seeMessage()
 
         return $this;
     }
+
+    protected function dontSeeMessage()
+    {
+        $this->assertTrue($this->getMessages()->isEmpty());
```

Comment:
@alepeino Good shout! Yep, that's why you would need it fluent! I'll add the changes laters today.

Thanks